### PR TITLE
Updates to General Bug Report template

### PR DESCRIPTION
**Pull request by pseudoyim**
- Make wording more concise throughout the template.
- Add section for user to indicate the impacted product.
- Add section for user to describe the expected behavior.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -57,7 +74,7 @@ body:
     attributes:
       label: Conda config
       description: |
-        Let's see what your Conda config looks like. Run the following command from your command line and paste the output
+        Let's examine your conda configurations. Run the following command and paste the output in the field below.
 
         ```bash
         conda config --show-sources
```
Comment (pseudoyim) [inline .github/ISSUE_TEMPLATE/bug_report.yml:80]: Should this command actually be `conda config --show`?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -1,52 +1,69 @@
...
+      placeholder: Tell us what you expected to happen.
     validations:
       required: true
   - type: textarea
     id: info
     attributes:
       label: Conda info
       description: |
-        Let's get the info on your conda setup. Run the following command from your command line and paste the output
+        Let's examine your conda installation. Run the following command and paste the output in the field below.
```
[the remark this answers is not in the document]
Comment (pseudoyim) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I'll just remove the word "conda" from this sentence (and the others) to make is easier to read. If they're opening an issue here, it's clear we're talking about their installation of one of our products.

**Review by skalburgi — COMMENTED** [review]
- in description: Should 'our products' be named perhaps Conda/Anaconda Distribution or any other relevant name?
- also, in the expected behavior section or in the 'tell us what happened' section (or any other relevant place), steps to reproduce the issue would be helpful to include. or something on the lines 'Please include steps to reproduce and any relevant error/(s), if available.'

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (csoja) [reply .github/ISSUE_TEMPLATE/bug_report.yml:80]: conda config --show-sources  provides what is in the various .condarc files - which is often very helpful in tracking down a problem.  
conda config --show is also helpful though, I could see an argument to include both - but I think usually just the contents of the .condarc files is often enough to help debug someone's issue.

**Comment by csoja** [comment]
> 

In this repository, we don't want issues for conda - only for Anaconda Navigator, the installers, perhaps anaconda.org.  Conda issues should be logged in the conda repository.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -1,52 +1,69 @@
 name: General Bug Report
-description: Create a report to help us improve
+description: Report a bug or other issue to help us improve our products.
```
Comment (csoja) [inline .github/ISSUE_TEMPLATE/bug_report.yml:2]: our products => the Anaconda Installers, Anaconda Navigator, Anaconda built packages and Anaconda.org.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -1,52 +1,69 @@
...
+      label: Impacted product
+      description: Please indicate the product(s) for which you are reporting the bug or issue.
+      options:
+        - label: Anaconda Distribution (formerly Anaconda Individual Edition)
+          required: false
+        - label: Miniconda
+          required: false
+        - label: A conda package
+          required: false
+
```
Comment (csoja) [inline .github/ISSUE_TEMPLATE/bug_report.yml:44]: Anaconda Navigator should also be listed here.  
Perhaps also Anaconda.org - I don't know where else you submit an issue for Anaconda.org - there used to be another repository, but it was shut down a long time ago.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -1,52 +1,69 @@
...
+    id: product
+    attributes:
+      label: Impacted product
+      description: Please indicate the product(s) for which you are reporting the bug or issue.
+      options:
+        - label: Anaconda Distribution (formerly Anaconda Individual Edition)
+          required: false
+        - label: Miniconda
+          required: false
+        - label: A conda package
```
Comment (csoja) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]: I think this should be An Anaconda-built conda package  - we don't want issue reports for packages built by conda-forge, etc.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (pseudoyim) [reply .github/ISSUE_TEMPLATE/bug_report.yml:80]: I see. Thanks @csoja

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (pseudoyim) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Thanks @csoja. Navigator has [their own form](https://github.com/ContinuumIO/anaconda-issues/issues/new?assignees=&labels=type-bug%2Cpkg-navigator&template=navigator_bug_report.yml), but I added the others.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (pseudoyim) [reply .github/ISSUE_TEMPLATE/bug_report.yml:46]: Navigator has [their own form](https://github.com/ContinuumIO/anaconda-issues/issues/new?assignees=&labels=type-bug%2Cpkg-navigator&template=navigator_bug_report.yml).
Added Anaconda.org.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 5 above)
Comment (pseudoyim) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Done. Thank you!

**Review by skalburgi — APPROVED** [review]
LGTM

**Pull request merged**